Update tutorial-classifying-iris-part-1.md

jainr opened the pull request:
Add datasource was missing a step to "select the finish button" to complete the process.

azurecla commented:
Hi __@ZainRizvi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (zarizvi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ZainRizvi commented:
Thanks Jim. About how long does it take for updated documentation to show up on Azure?

jainr committed:
Update tutorial-classifying-iris-part-1.md

Add datasource was missing a step to select the finish button to complete the process.
